lib/data/block.go

```diff
@@ -95,20 +95,24 @@ func (block *Block) Read(serverInfo *ServerInfo, decoder *binary.Decoder) (err e
 	return nil
 }
 
+func (block *Block) writeLen(l, num, level int) {
```

Comment:
use a more intuitive name instead of `l`?

Reply:
I changed it